TPSDropDown/Classes/TPSDropDown.h

```diff
@@ -26,11 +26,16 @@
 
 @protocol TPSDropDownItem <NSObject>
 
-@required
+@optional
```

Comment:
I think we should leave @required specification if we don't check -respondsToSelector inside TPSDropDown.